mm2src/mm2_main/src/rpc.rs

```diff
@@ -48,8 +48,9 @@ mod dispatcher_legacy;
 #[path = "rpc/lp_commands/lp_commands_legacy.rs"]
 pub mod lp_commands_legacy;
 mod rate_limiter;
+#[path = "rpc/wc_commands/wc_commands.rs"] pub mod wc_commands;
```

Comment:
can we rename `wc_commands.rs` -> `mod.rs` and not use the path attribute here?

Reply:
yes, done